test.e2e/spec/services.spec.js

```diff
@@ -1241,16 +1243,80 @@ define(['angular', 'given', 'util'], function(angular, given, util) {
           });
       });
 
-      it('defines the "Product" model as "lbServices.Product"', function() {
+      it('does not define "Product" model', function() {
         expect(function() {
           $injector.get('Product');
         }).to.throw(/Unknown provider/);
+      });
+      it('defines the "Product" model as "lbServices_Product"', function() {
         expect(function() {
           $injector.get('lbServices_Product');
         }).to.not.throw();
       });
     });
 
+    describe('$resource generated with namespaceCommonModels:true and ' +
+      'default namespaceDelimiter:.', function() {
+      it('fails to find "Auth" common model', function() {
+        given.servicesForLoopBackApp({
```

Comment:
Missing `return`. As a result, the failure of this promise chain is recognised by mocha while running a different test. After I added `return`, the test is failing with the following error:

```
      ✖ fails to find "Auth" common model
ERROR: 'undefined is not a constructor (evaluating 'window.Mocha.utils.stringify(error_.actual)')
http://localhost:9876/base/node_modules/karma-mocha/lib/adapter.js?9112a0a88c794e8094a758094ffc9093543c3edd:30', 'Possibly unhandled rejection: {"line":30,"sourceURL":"http://localhost:9876/base/node_modules/karma-mocha/lib/adapter.js?9112a0a88c794e8094a758094ffc9093543c3edd","stack":"processAssertionError@http://localhost:9876/base/node_modules/karma-mocha/lib/adapter.js?9112a0a88c794e8094a758094ffc9093543c3edd:30:50\nhttp://localhost:9876/base/node_modules/karma-mocha/lib/adapter.js?9112a0a88c794e8094a758094ffc9093543c3edd:97:49\nemit@http://localhost:9876/base/node_modules/mocha/mocha.js?3d9e18d0ef22ae6ebab0de08c114ab08cebcaf8a:583:18\nfail@http://localhost:9876/base/node_modules/mocha/mocha.js?3d9e18d0ef22ae6ebab0de08c114ab08cebcaf8a:4554:12\nhttp://localhost:9876/base/node_modules/mocha/mocha.js?3d9e18d0ef22ae6ebab0de08c114ab08cebcaf8a:4810:20\ndone@http://localhost:9876/base/node_modules/mocha/mocha.js?3d9e18d0ef22ae6ebab0de08c114ab08cebcaf8a:4300:7\nprocessQueue@http://localhost:9876/base/node_modules/angular/angular.js?a03f84c29b4caaa89137d54c47b79d2b999f5ee7:16832:39\nhttp://localhost:9876/base/node_modules/angular/angular.js?a03f84c29b4caaa89137d54c47b79d2b999f5ee7:16876:39\n$digest@http://localhost:9876/base/node_modules/angular/angular.js?a03f84c29b4caaa89137d54c47b79d2b999f5ee7:17971:17\nhttp://localhost:9876/base/node_modules/angular/angular.js?a03f84c29b4caaa89137d54c47b79d2b999f5ee7:18200:33\ncompleteOutstandingRequest@http://localhost:9876/base/node_modules/angular/angular.js?a03f84c29b4caaa89137d54c47b79d2b999f5ee7:6274:15\nhttp://localhost:9876/base/node_modules/angular/angular.js?a03f84c29b4caaa89137d54c47b79d2b999f5ee7:6554:33"}'
```